Makefile

```diff
@@ -122,9 +122,9 @@ CUOBJ = $(patsubst %.cu, build/%_gpu.o, $(CUSRC))
 
 # extra operators
 ifneq ($(EXTRA_OPERATORS),)
-	EXTRA_SRC = $(wildcard $(patsubst %, %/*.cc %/*/*.cc, $(EXTRA_OPERATORS)))
+	EXTRA_SRC = $(wildcard $(foreach DIR, $(EXTRA_OPERATORS), $(foreach PAT, *.cc */*.cc, $(DIR)/$(PAT))))
```

Comment:
I think foreach is gnu specific and doesn't work on other make implementations?

Plus the current makefile already supports multiple extra operators. just separate them by space